src/main/java/hudson/plugins/jobConfigHistory/JobConfigHistory.java

```diff
@@ -99,7 +104,7 @@ public class JobConfigHistory extends Plugin {
     /**
      * Regular expression pattern for 'system' configuration files to exclude from saving.
      */
-    private String excludePattern;
+    private String excludePattern = JobConfigHistoryConsts.DEFAULT_EXCLUDE;

```

Comment:
The mechanism to set the default value for `excludePattern` has changed. It is not any longer set via `default="${it.defaultExcludePattern}"` in config.jelly. Should the method `getDefaultExcludePattern()` therefore be removed too?

Reply:
Right.